descriptors/ModuleDescriptor-template.json

```diff
@@ -122,6 +122,18 @@
           "modulePermissions": []
         }
       ]
+    },
+    {
+      "id": "marc",
```

Comment:
Hi @askhat-abishev,

In Okapi, each block in the "providers" array represent an interface. So, here you have defined an interface named "marc", which is implemented by mod-linked-data module.

Per okapi documentation [https://github.com/folio-org/okapi/blob/master/doc/guide.md](https://github.com/folio-org/okapi/blob/master/doc/guide.md):

```
Note that we can have multiple modules providing the same interface. These can be deployed in Okapi simultaneously, but only one such module can be enabled for any given tenant at a given time
```
This is little strange. If there is another module which has a provider with id="marc", then linked-data module cannot be enabled along with that module per this documentation.

There is a good chance that another module, which has a "provider" with "id=marc" already exists. So, I think it is better to create this API under the existing "resource" interface.

May be some thing like `/resource/inventory-id/{id}/import-check`. I know this is not a good API URL. I can't think about anything better now :)

Reply:
done